src/awake.hpp

```diff
@@ -209,14 +209,26 @@
 #define RACE_SATYR                 18
 #define RACE_NIGHTONE              19
 #define RACE_DRYAD                 20
-#define RACE_DRAGON                21
-#define RACE_ELEMENTAL             22
-#define RACE_SPIRIT                23
-#define RACE_PC_CONJURED_ELEMENTAL 24
-#define NUM_RACES                  24  /* This must be the NUMBER of races */
+#define RACE_WESTERN_DRAGON        21
+#define RACE_EASTERN_DRAGON        22
+#define RACE_FEATHERED_SERPENT     23
+#define RACE_DRAKE_HUMAN           24
+#define RACE_DRAKE_DWARF           25
+#define RACE_DRAKE_ELF             26
+#define RACE_DRAKE_ORK             27
+#define RACE_DRAKE_TROLL           28
+#define RACE_GHOUL_HUMAN           29
+#define RACE_GHOUL_DWARF           30
+#define RACE_GHOUL_ELF             31
+#define RACE_GHOUL_ORK             32
+#define RACE_GHOUL_TROLL           33
+#define RACE_ELEMENTAL             34
+#define RACE_SPIRIT                35
+#define RACE_PC_CONJURED_ELEMENTAL 36
```

Comment:
The `RACE_ELEMENTAL`, `RACE_SPIRIT`, and `RACE_PC_CONJURED_ELEMENTAL` definitions shouldn't be changed, because doing so will change the races of all existing elementals / spirits / etc (they would become feathered serpents, human drakes, and dwarven drakes respectively).

Note also that all current `RACE_DRAGON` mobs (if there are any) would become Western dragons with this change, but I think that part is acceptable?

Reply:
I see why you moved them though, since the "maximum valid player race" define kicks in there. The easier solution is probably to just remove the `X_VALID_PLAYER_RACE` definitions and convert the code that uses them from if-checks / for-loops to switch statements.

Reply:
I actually tested this since I thought the same thing when writing it, but all pre-existing race type settings for mobs remained intact. Spirits are still Spirits, Elementals are still Elementals, PC Conjured Elementals are still PC Conjured Elementals. Just ran the code on my branch again to double check: Still using their old race type setting across the board

Reply:
Poked at it some more, but I'm pretty sure mobs save race definition type instead of race definition value

Reply:
Oh nice, I didn't realize that! Sounds good then.